build.sbt

```diff
@@ -2,7 +2,7 @@ name := "dfdl-mil-std-2045"
 
 organization := "com.owlcyberdefense"
 
-version := "1.0.4"
+version := "1.0.5"
```

Comment:
Sorry I didn't make enough noise about this dev branch.

There is now a dev branch here as well as a master branch. This is so that we can push the use of daffodil 3.4.0-SNAPSHOT here on the dev branch, and make changes that depend on 3.4.0-SNAPSHOT also. 

You will want to push this change to the dev branch.

Reply:
Changed target to dev branch